### PR TITLE
Add NonPayableContract and RejectingPaymentsContract scenarios

### PR DESCRIPTION
**Pull request by stoqnkpL**

### CODE REVIEW

File: test/LowLevelCall.ts [hunk 1]
```diff
@@ -205,18 +219,164 @@ describe('EVM Calls and internal calls edge cases test', function() {
...
+    
+    console.log("tx hash: ", tx.hash);
+
+    const rc = await tx.wait();
+
+    expect(rc.status).to.be.eq(1);
+  });
+
+  // EOA -call→ Caller -transfer→ NonPayableContract
+  it('should be able to make an internal TRANSFER to NonPayableContract', async function() {
```
Comment (IvanKavaldzhiev) [inline test/LowLevelCall.ts]: Probably `should not be able`?

File: test/LowLevelCall.ts [hunk 2]
```diff
@@ -205,18 +219,164 @@ describe('EVM Calls and internal calls edge cases test', function() {
...
+      // this throws an error because the transaction reverted
+      await tx.wait();
+
+    } catch(error: any) {
+      expect(error.message).to.match(/(?:transaction execution reverted|transaction reverted:)/);
+    };
+  });
+
+  // EOA -call→ Caller -transfer→ RejectingPaymentsContract
+  it('should be able to make an internal TRANSFER to RejectingPaymentsContract', async function() {
```
Comment (IvanKavaldzhiev) [inline test/LowLevelCall.ts]: Should not be able?
